src/TestClients/AnalyticsTestClient/Menus/ResultsApiConsumerMenu.cs

```diff
@@ -0,0 +1,107 @@
+
+
+                MenuItems.Add('1', new ConsoleMenuItem("DAU", () => GetResults(dauSerializer, "dau", "session-start")));
+                MenuItems.Add('2', new ConsoleMenuItem("Clustering", () => GetResults(clusteringSerializer, "clustering", "geo-location")));
+            }
+
+            private void GetResults(IMessageFormatter formatter, string pipeline, string messageType)
+            {
+                var start = GetDate("Start date");
+                var end = GetDate("End date");
+
+                if (start == null || end == null) return;
+
+                var filePathAlgorithm = new PipelineDateFilePathAlgorithm(newFileOption: NewFileNameOptions.Every5Minutes);
+
+                IFilePathAlgorithm _filePathAlgorithm = new PipelineDateFilePathAlgorithm(newFileOption: NewFileNameOptions.Every5Minutes);
+
+                Console.Write("Root directory: ");
+
+                var path = Console.ReadLine();
+
+                var f = new Nether.Analytics.FileResultsReader(formatter, _filePathAlgorithm, path, pipeline, messageType);
+                foreach (var msg in f.Get(start.Value, end.Value))
+                {
+                    Console.WriteLine(msg.ToString());
+                }
+            }
+
+            private DateTime? GetDate(string type)
+            {
+                Console.Write($"{type}:");
+                var d = Console.ReadLine();
+
+                DateTime date;
+                if (!DateTime.TryParse(d, out date))
+                {
+                    Console.ForegroundColor = ConsoleColor.Red;
+                    Console.WriteLine("Date was incorrectly formatted.");
+                    Console.ResetColor();
+                    return null;
+                }
+
+                return date;
+            }
+        }
+
+
+        private class ResultsFileSystemMenu : ConsoleMenu
+        {
+            public ResultsFileSystemMenu()
+            {
+                Title = "Latest Results (FS)";
+
+                var dauSerializer = new CsvMessageFormatter("id", "type", "version", "enqueuedTimeUtc", "gameSession", "gamerTag");
+                var clusteringSerializer = new CsvMessageFormatter("id", "type", "version", "enqueueTimeUtc", "gameSession", "lat", "lon", "geoHash", "geoHashPrecision", "geoHashCenterLat", "geoHashCenterLon", "rnd");
+
+
+                MenuItems.Add('1', new ConsoleMenuItem("DAU", () => GetLatestFromFileSystem(dauSerializer, "dau", "session-start")));
+                MenuItems.Add('2', new ConsoleMenuItem("Clustering", () => GetLatestFromFileSystem(clusteringSerializer, "clustering", "geo-location")));
+            }
+
+            public void GetLatestFromFileSystem(IMessageFormatter formatter, string pipeline, string messageType)
+            {
+                IFilePathAlgorithm _filePathAlgorithm = new PipelineDateFilePathAlgorithm(newFileOption: NewFileNameOptions.Every5Minutes);
```

Comment:
Don't use underscore as a prefix for naming local variables (only for private class fields)